.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: Provide a general description of the bug, in plain text. Use the dedicated fields below for code snippets.
+      placeholder: Description of the bug
+    validations:
+      required: true
+  - type: textarea
+    id: java-input
+    attributes:
+      label: Source code you are trying to analyze/transform
+      description: Provide a minimal example of source code you are trying to analyze/transform that causes the bug to appear.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: spoon-code
+    attributes:
+      label: Source code for your Spoon processing
+      description: Provide your processing code that uses Spoon and causes the bug to appear. This will be automatically formatted, so no need for backticks.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: output
+    attributes:
+      label: Output
+      description: The output that spoon provides, or a stacktrace if it crashes. This will be automatically formatted, so no need for backticks.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: expected-output
+    attributes:
+      label: Expected output
+      description: If you know what kind of output you expect to get, please provide it here. This will be automatically formatted, so no need for backticks.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: version
+    attributes:
+      label: Spoon Version
```

Comment:
Maybe also provide a "java compliance level" or something field? Not sure if that is helpful enough, we don't want to burden the user too much.

Reply:
It's not a required input for using Spoon, so I think we shouldn't include that. I wanted to put in something like "noclasspath", but that's also not required input so I left it out. Like you said, we don't want to make it too much of a hassle to report a problem.